pdseg/tools/create_dataset_list.py

```diff
@@ -130,12 +130,12 @@ def generate_list(args):
             for item in range(num_images):
                 left = image_files[item].replace(dataset_root, '')
                 if left[0] == os.path.sep:
-                    left = left.lstrip(os.path.sep)
+                    left = left.lstrip(os.path.sep).replace('\\', '/')
```

Comment:
@Sunyingbin thanks for your pr.Is there any problem with this code? We have tested this code on win10 and it behaves normally.

Reply:
I’m not sure, because this code is running wrong on my win10 , You can try a few more times.

Reply:
I use PaddleX's Deeplabv3+ for trainning, Prompt can not find the image path.

Reply:
Can you paste the error stack? Also, what dataset do you use?

Reply:
I user my own dataset
train_dataset = pdx.datasets.SegDataset(
    data_dir='waterTmp',
    file_list='waterTmp/train.txt',
    label_list='waterTmp/class_names.txt',
    transforms=train_transforms,
    shuffle=True)
eval_dataset = pdx.datasets.SegDataset(
    data_dir='waterTmp',
    file_list='waterTmp/val.txt',
    label_list='waterTmp/class_names.txt',
    transforms=eval_transforms)
here is the error:
---------------------------------------------------------------------------OSError                                   Traceback (most recent call last)<ipython-input-6-5759cf7f127c> in <module>
      4     label_list='waterTmp/class_names.txt',
      5     transforms=train_transforms,
----> 6     shuffle=True)
      7 eval_dataset = pdx.datasets.SegDataset(
      8     data_dir='waterTmp',
/opt/conda/envs/python35-paddle120-env/lib/python3.7/site-packages/paddlex/cv/datasets/seg_dataset.py in __init__(self, data_dir, file_list, label_list, transforms, num_workers, buffer_size, parallel_method, shuffle)
     71                 if not osp.exists(full_path_im):
     72                     raise IOError(
---> 73                         'The image file {} is not exist!'.format(full_path_im))
     74                 if not osp.exists(full_path_label):
     75                     raise IOError('The image file {} is not exist!'.format(
OSError: The image file waterTmp/images\train\20200529093257.png is not exist!

Reply:
I noticed the path information in your error stack
```
/opt/conda/envs/python35-paddle120-env/lib/python3.7/site-packages/paddlex/cv/datasets/seg_dataset.py
```
It seems that you have generated the dataset in the windows environment and then trained in the linux environment?

Reply:
yes，I have generated the dataset in the windows environment and then trained on the Baidu AIStudio。

Reply:
This leads to the presence of '\' and '/' separators in the path of the final input training picture, resulting in the system not being able to correctly recognize the path. It is recommended that you perform data processing and training in the same system environment.

Reply:
I think generated the dataset in the windows environment and then trained on the Baidu AIStudio environment  is very common, '\' separators should not exist in textFile

Reply:
When processing data, the root path of the data set and the image path need to be spliced, so the separator must be stored in the file. In addition, it seems to be crashed in paddlex, this problem needs to be fixed by paddlex.